app/pods/forgot-password/controller.js

```diff
@@ -0,0 +1,32 @@
+import Controller from '@ember/controller';
+import { dropTask } from 'ember-concurrency-decorators';
+import { inject as service } from '@ember/service';
+import { tracked } from '@glimmer/tracking';
+
+export default class ForgotPasswordController extends Controller {
+  @service api;
+
+  @tracked isError = false;
+  @tracked isSuccess = false;
+
+  formData = {
+    email: ''
+  }
+
+  @dropTask
+  *forgotPasswordTask() {
```

Comment:
we can also make this a pure function but accepting the argument of email from the frontend, but it won't be necessary as the variable lives inside the same class